src/artemis/ispyb/store_in_ispyb.py

```diff
@@ -0,0 +1,133 @@
+import ispyb
+from sqlalchemy.connectors import Connector
+
+from src.artemis.ispyb.ispyb_dataclasses import GridInfo, DataCollection, DataCollectionGroup, Position
+
+
+ISPYB_CONFIG_FILE = "/dls_sw/dasc/mariadb/credentials/ispyb-dev.cfg"
+
+
+class StoreInIspyb:
+
+    conn: Connector
```

Comment:
You have to be careful in Python about instance variables vs class variables (https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables). This should probably be an instance variable.